smoke_test/mail_box.rb

```diff
@@ -0,0 +1,35 @@
+require 'mail'
+require 'net/imap'
+
+module SmokeTest
+  module MailBox
+    SMOKE_TEST_EMAIL_PASSWORD = ENV.fetch 'SMOKE_TEST_EMAIL_PASSWORD'
+    SMOKE_TEST_EMAIL_HOST = ENV.fetch 'SMOKE_TEST_EMAIL_HOST'
```

Comment:
Ahhh I'm such a dufus.. so upon closer inspection these are different constants to what `SmokeTest` offers :sob: